internal/engine/interfaces/interface.go

```diff
@@ -76,6 +76,10 @@ const (
 	ActionOptUnknown
 )
 
+func (a ActionOpt) String() string {
+	return [...]string{"on", "off", "dry_run", "unknown"}[a]
```

Comment:
what is this witchery

Reply:
it just says that the size of the array is determined by the number of the elements